amp.manifest

```diff
@@ -0,0 +1,25 @@
+versionAmp           = latest
+versionInfluxDB      = 1.1.3
+versionKapacitor     = 1.1.3
+versionTelegraf      = 1.1.2
+versionGrafana       = 1.1.3
+versionElasticsearch = 5.1.2
+versionEtcd          = 3.1.0
+versionNats          = v0.3.8
+versionHaproxy       = 1.0.5
```

Comment:
the pb is we now have many images in the same repo, each with different lifecycles. If we have to tag and release all the images each time we have a fix on one image, it's going to be a mess. A way to do it: https://github.com/appcelerator/amp/issues/711#issuecomment-275542256